svgcheck/run.py

```diff
@@ -107,7 +108,10 @@ def main():
         wp.color_threshold = options.grey_level
 
     if len(args) < 1:
-        source = os.getcwd() + "/stdin"
+        with tempfile.NamedTemporaryFile(dir=options.cache, delete=False, mode="w+t") as tmp_file:
```

Comment:
This leaves the tempfile around after the job is complete - is that _intentional_? It's not clear how leaving such things in the cache directory will help. Consider a different place than the cache directory for them in any case - they are a different kind of thing.

Reply:
This has been addressed with 01c7a0e69d7d3e3a03d3ad6a063f698ef6fcb27d and 0ba8df89638ea25d45ab9c5fbc09c0b83a06d7cb